assets/src/edit-story/elements/index.js

```diff
@@ -34,7 +34,10 @@ import * as videoElement from './video';
 
 export const createNewElement = (type, attributes = {}) => {
   const element = elementTypes.find((el) => el.type === type);
-  const defaultAttributes = element ? element.defaultAttributes : {};
+  if (!element) {
+    throw new Error(`Unknown element type: ${type}`);
```

Comment:
Perhaps we should also throw an error if a Page is inserted without background element id? E.g if someone uses `createNewElement` directly instead of going through `createPage`. That would also cause all sorts of weirdness such as the history breaking ;)

Reply:
Hm, we could, but then we ought to check for all required attributes for all types of elements (non-pages without `x` or `y` would break stuff too). I think it's okay for now.